remotespark/utils/constants.py

```diff
@@ -9,6 +9,10 @@
 SESSION_KIND_SPARKR = "sparkr"
 SESSION_KINDS_SUPPORTED = [SESSION_KIND_SPARK, SESSION_KIND_PYSPARK, SESSION_KIND_SPARKR]
 
+SESSION_CREATION_START_EVENT = "notebookSessionCreationStart"
+SESSION_CREATION_END_EVENT = "notebookSessionCreationEnd"
+EVENTS = {SESSION_CREATION_START_EVENT, SESSION_CREATION_END_EVENT}
```

Comment:
For newly added events, this should be reflected here instead of adding it in the SparkEvents file.